src/device-registry/.env.production.template

```
@@ -196,4 +196,7 @@ PROD_ADMIN_SETUP_SECRET=
 PROD_HCAPTCHA_SECRET_KEY=
 
 # Environment Defaults
-PROD_DEFAULT_COHORT=
\ No newline at end of file
+PROD_DEFAULT_COHORT=
+# Comma-separated cohort names that cannot be assigned to users/groups (e.g. "airqo,another_system_cohort").
+# "airqo" is always blocked by default even without this variable.
+PROD_PROTECTED_COHORT_NAMES=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)